docs/docs-app/components/layout.js

```diff
@@ -17,12 +18,9 @@ class Layout extends Component {
     } = this.props;
 
     return (
-      <div className="layout-container">
-        <Header />
-        <div className="main-content">
-          <TableOfContents pages={pages} parentRoute={path}/>
-          { {...children, props: {...children.props, markdownPages}} }
-        </div>
+      <div className="f fg">
```

Comment:
Simple util classes

f -> flex
fg -> flex-grow: 1

You can change it if you like, I think it could be a could thing instead of repeating rules in specific classes. That's more an attempt that anything for now